tutorials/plugins/editor/inspector_plugins.rst

```diff
@@ -123,7 +123,7 @@ specifically add :ref:`class_EditorProperty`-based controls.
         return true
 
 
-    func _parse_property(object, type, path, hint, hint_text, usage):
+    func _parse_property(object, type, path, hint, hint_text, usage, wide):
```

Comment:
```suggestion
    func _parse_property(object, type, name, hint_type, hint_string, usage_flags, wide):
```
To match the default, and matches the C#

Reply:
Looking at the [underlying engine code](https://github.com/godotengine/godot/blob/4b7cc99bc195387c784efced7fdd360501d9a5cf/editor/editor_inspector.cpp#L1098), the engine names are being remapped from the the internal names with the original names and type being

```c++
(Object *p_object, const Variant::Type p_type, const String &p_path, const PropertyHint p_hint, const String &p_hint_text, const BitField<PropertyUsageFlags> p_usage, const bool p_wide)
...
GDVIRTUAL_CALL(_parse_property, p_object, p_type, p_path, p_hint, p_hint_text, p_usage, p_wide, ret);
```
and the binding
```c++
GDVIRTUAL_BIND(_parse_property, "object", "type", "name", "hint_type", "hint_string", "usage_flags", "wide")
```

Is there a standard for names used in gdscript?  Looking around I don't see other instances of type information being added to the name (`hint` -> `hint_type`, `usage` -> `usage_flags`) or rephrasing (`hint_text` -> `hint_string`).  

Also `path` was renamed to `name`.   It does appear to be the local property name rather than a path though.

Reply:
The binding should be used here, as it is what you get if you add this method in script, IMO

Reply:
I was more questioning whether there was also an issue with the binding or not. 

I just need to change the call in the method to match now.